doc/sphinx/proofs/writing-proofs/rewriting.rst

```diff
@@ -395,35 +395,48 @@ conclusion.
 For backward compatibility, the notation :n:`in {+ @ident}` performs
 the conversion in hypotheses :n:`{+ @ident}`.
 
-.. tacn:: cbv {? @strategy_flag }
-          lazy {? @strategy_flag }
-   :name: cbv; lazy
-
-   These parameterized reduction tactics apply to any goal and perform
-   the normalization of the goal according to the specified flags. In
-   correspondence with the kinds of reduction considered in Coq namely
-   :math:`\beta` (reduction of functional application), :math:`\delta`
-   (unfolding of transparent constants, see :ref:`vernac-controlling-the-reduction-strategies`),
-   :math:`\iota` (reduction of
-   pattern matching over a constructed term, and unfolding of :g:`fix` and
-   :g:`cofix` expressions) and :math:`\zeta` (contraction of local definitions), the
-   flags are either ``beta``, ``delta``, ``match``, ``fix``, ``cofix``,
-   ``iota`` or ``zeta``. The ``iota`` flag is a shorthand for ``match``, ``fix``
-   and ``cofix``. The ``delta`` flag itself can be refined into
-   :n:`delta [ {+ @qualid} ]` or :n:`delta - [ {+ @qualid} ]`, restricting in the first
-   case the constants to unfold to the constants listed, and restricting in the
-   second case the constant to unfold to all but the ones explicitly mentioned.
+.. tacn:: lazy {? @strategy_flag } {? @occurrences }
+          cbv {? @strategy_flag } {? @occurrences }
+
+   .. todo PR: should this explanation move to where strategy_flag is defined?
+      or move that syntax definition here?
+
+   Normalize the goal as specified by the :n:`@strategy_flag`\s. The flags
+   are:
+
+   `beta` (:math:`\beta`)
+     reduction of functional application
+
+   `delta` (:math:`\delta`)
+     unfolding of transparent constants, see :ref:`vernac-controlling-the-reduction-strategies`
+
+   `iota` (:math:`\iota`)
+     reduction of pattern matching (`match`) over a constructed term and unfolding of :g:`fix` and
+     :g:`cofix` expressions.  Shorthand for `match fix cofix`
+
+   `zeta` (:math:`\zeta`)
+      contraction of local definitions
```

Comment:
I did this, with some caveats:
- I tried to make the section titles also be the term definitions, but it converted an explicit link name of `beta-reduction` to just `reduction`, giving a naming conflict.  It also put the title in italic, which I could live with.
- The link targets I added are in the middle of the text, a bit less than ideal.
- The target text is not very clear at all.  Maybe now is a good time to provide a much better description--though that will probably take us a couple go-rounds.
- The glossary lists these spelled out, e.g. `beta-reduction`.  I figured it would not be easy to find them starting with a Greek letter.  I suppose it's possible to create 2 glossary entries, one with Greek and one only Roman.  FWIW, I had to look up zeta, the others I knew.

Reply:
>The target text is not very clear at all. Maybe now is a good time to provide a much better description--though that will probably take us a couple go-rounds.

Sure, we can try that.

> I suppose it's possible to create 2 glossary entries, one with Greek and one only Roman.

I think Roman only is fine.